### PR TITLE
Seasonal: Balance Update

### PR DESCRIPTION
**Pull request by cadaeic**
- Buff Anttya
- Buff starry signature move
- Buff Hashtag
- Buff Steamroll
- Nerf Haund

### CODE REVIEW

File: mods/seasonal/scripts.js [hunk 1]
```diff
@@ -881,6 +881,8 @@ exports.BattleScripts = {
 		let pool = Object.keys(sets);
 		for (let i = 0; i < 6; i++) {
 			let name = this.sampleNoReplace(pool);
+			if (i === 0) name = 'Ascriptmaster';
+			if (i === 1) name = 'Death on Wings';
```
Comment (Marty-D) [inline mods/seasonal/scripts.js]: I'm guessing these two lines were just for testing?

Comment (cadaeic) [inline mods/seasonal/scripts.js]: Eek! I forgot to remove those ;~;

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -2574,7 +2575,7 @@ exports.BattleMovedex = {
 					const thisSide = this.sides[s];
 					for (let p in thisSide.active) {
 						const pokemon = thisSide.active[p];
-						if ((pokemon.types[0] === 'Flying' && !pokemon.types[1]) || !pokemon.hp) continue;
+						if ((pokemon.template.types[0] === 'Flying' && !pokemon.template.types[1]) || !pokemon.hp) continue;
 						pokemon.setType(pokemon.template.types, true);
```
Comment (Zarel) [inline mods/seasonal/moves.js]: This seems like it would still be wrong?

Comment (Zarel) [inline mods/seasonal/moves.js]: `setType` takes a string naming a single type, not a list of types.

Comment (Zarel) [inline mods/seasonal/moves.js]: That's what @urkerab was trying to fix in #2515

Comment (Zarel) [inline mods/seasonal/moves.js]: This line still needs to be replaced with `pokemon.types = pokemon.template.types;`

**Comment by Zarel** [comment]
Just so you don't forget:

Also, I am deprecating `.setType(type, true)`. Please use `.types = [type]` instead.

**Comment by cadaeic** [comment]
Fixed, it now accesses types directly, and checks against `pokemon.template.types` instead.

**Comment by Zarel** [comment]
I have no objections. @Joimer?

**Comment by Joimer** [comment]
Seems good to go!

**Comment by Zarel** [comment]
There's still a lot of instances of `.setType(..., true)` in `mods/seasonal/`. Can you fix those?

**Comment by cadaeic** [comment]
Will fix in next patch.

**Pull request merged**